docs/application/native/guides/text-input/voice-control-manager.md

```diff
@@ -0,0 +1,1134 @@
+    > The `VC_COMMAND_TYPE_EXCLUSIVE` commands have special priority, so these are used when recording starts with `vc_mgr_start(true)`.
+    > When recording starts with `vc_mgr_start(true)`, commands having other priorities are not recognized, only `VC_COMMAND_TYPE_EXCLUSIVE` commands are recognized.
+
+    ```c
+    void
+    create_command_list()
+    {
+        int ret;
+        vc_cmd_list_h vc_cmd_list;
+        ret = vc_cmd_list_create(&vc_cmd_list);
+        if (VC_ERROR_NONE != ret)
+            /* Error handling */
+    }
+    ```
+
+2.  Create a command.
+    Create a command handle, and then define the command and type:
+
+    ```c
+    void
+    create_command()
+    {
+        int ret;
+        vc_cmd_h vc_cmd = NULL;
+        ret = vc_cmd_create(&vc_cmd);
+        if (VC_ERROR_NONE != ret)
+            /* Error handling */
+
+        ret = vc_cmd_set_command(vc_cmd, "open");
+        if (VC_ERROR_NONE != ret)
+            /* Error handling */
+
+        ret = vc_cmd_set_type(vc_cmd, VC_COMMAND_TYPE_SYSTEM);
+        if (VC_ERROR_NONE != ret)
+            /* Error handling */
+    }
+    ```
+
+3.  Add the command to the command list:
+
+    ```c
+    void
+    add_command(vc_cmd_list_h vc_cmd_list, vc_cmd_h vc_cmd)
+    {
+        int ret;
+        ret = vc_cmd_list_add(vc_cmd_list, vc_cmd);
+        if (VC_ERROR_NONE != ret)
+            /* Error handling */
+    }
+    ```
+
+    If necessary, you can also remove commands from the command list. After you remove command and you no longer use it, destroy the command handle with `vc_cmd_destroy()`:
+
+    ```c
+    void
+    remove_command(vc_cmd_list_h vc_cmd_list, vc_cmd_h vc_cmd)
+    {
+        int ret;
+        ret = vc_cmd_list_remove(vc_cmd_list, vc_cmd);
+        if (VC_ERROR_NONE != ret)
+            /* Error handling */
+
+        /* Destroy the command if no longer needed */
+        ret = vc_cmd_destroy(vc_cmd);
+        if (VC_ERROR_NONE != ret)
+            /* Error handling */
+    }
+    ```
+
+    To retrieve the commands you have added to the command list:
+
+    - Use `vc_cmd_list_foreach_commands()` to get each command within a callback function:
+
+      ```c
+      bool
+      __vc_get_command_cb(vc_cmd_h vc_command, void* user_data)
+      {
+          /* Your code */
+
+          return true;
+      }
+
+      void
+      foreach_command(vc_cmd_list_h vc_cmd_list)
+      {
+          int ret;
+          vc_cmd_h vc_cmd;
+          ret = vc_cmd_list_foreach_commands(vc_cmd_list, __vc_get_command_cb, NULL);
+          if (VC_ERROR_NONE != ret)
+              /* Error handling */
+      }
+      ```
+
+    - Use `vc_cmd_list_get_current()` to get the current command in an output parameter:
+
+      > [!NOTE]
+      > When you get the command handle with `vc_cmd_list_get_current()`, do not release it. To release the command handle, call `vc_cmd_list_remove()` before `vc_cmd_destroy()`.
```

Comment:
```suggestion
      > [!NOTE]
      > When you get the command handle with `vc_cmd_list_get_current()`, you must not release it. Instead to release the command handle, use `vc_cmd_list_remove()` before `vc_cmd_destroy()`.

Reply:
The command handle from `vc_cmd_list_get_current()` is still in the command list, so if user release the handle using `vc_cmd_destroy()`, the command list can occur problem.
So, if user want to release the command handle, we recommend that first remove the command handle from list using `vc_cmd_list_remove()` and then release the commad handle using `vc_cmd_destroy()`.

So I think 'instead' is not right we intednded. How about this?
```suggestion
      > [!NOTE]
      > When you get the command handle with `vc_cmd_list_get_current()`, you must not release it. If you want to release the command handle, use `vc_cmd_list_remove()` before `vc_cmd_destroy()`.
```

Reply:
How about this? Although your suggestion is good too.
```suggestion
      > [!NOTE]
      > When you get the command handle with `vc_cmd_list_get_current()`, you must not release it, as releasing it directly may cause an error. To release the command handle, use `vc_cmd_list_remove()`, which removes the commands from the command list followed by  `vc_cmd_destroy()` to destroy the command list.

Reply:
Thanks.
I think your suggestion provides more detail information. I fix the description as yours.